pynest/examples/pynest-microcircuit/helpers.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+#
+# helpers.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+'''
+pynest microcicuit helpers
+----------------------------------------------
+
+Helpers file for the microcircuit.
+
+Hendrik Rothe, Hannah Bos, Sacha van Albada; May 2016
+'''
+
+import numpy as np
+from network_params import *
+from stimulus_params import *
+
+
+def get_weight(PSP_val):
+    """This function computes the weight which elicits a change in the membrane
+    potential of size PSP_val.
+
+    Arguements:
+    PSP_val: evoked postsynaptic potential
+
+    Returns: weight value
+
+    """
+    C_m = net_dict['n_params']['C_m']
+    tau_m = net_dict['n_params']['tau_m']
+    tau_ex = net_dict['n_params']['tau_ex']
+    tau_in = net_dict['n_params']['tau_in']
+
+    PSC_e_over_PSP_e = (((C_m)**(-1) * tau_m * tau_ex / (tau_ex - tau_m) * (
+        (tau_m / tau_ex)**(- tau_m / (tau_m-tau_ex)) - (
+            tau_m/tau_ex)**(- tau_ex/(tau_m-tau_ex))))**(-1))
+    PSC_e = (PSC_e_over_PSP_e * PSP_val)
+    return PSC_e
+
+
+def get_total_number_of_synapses():
+    """This function returns the total number of synapses between
+    all populations.
+
```

Comment:
Please make clear in the docstring, that this function is an internal helper function that ignores the `K_scaling` parameter.